improver/calibration/ensemble_calibration.py

```diff
@@ -449,12 +489,14 @@ def __init__(self, distribution, current_cycle, desired_units=None,
         Warns:
             ImportWarning: If the statsmodels module can't be imported.
         """
-        valid_distributions = (ContinuousRankedProbabilityScoreMinimisers().
-                               minimisation_dict.keys())
+        valid_distributions = (
+            ContinuousRankedProbabilityScoreMinimisers().minimisation_dict.keys()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [dict.keys referenced when not iterating](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5385606)